bot/utils.py

```diff
@@ -1,4 +1,3 @@
-# coding=utf-8
 
```

Comment:
Is PEP8 happy with these blank lines?

Reply:
I think by convention there should be no blank lines at the top of the file.

Reply:
sorted.